PWGHF/Core/HfMlResponseOmegacToOmegaPi.h

```diff
@@ -0,0 +1,433 @@
+  case static_cast<uint8_t>(InputFeaturesOmegacToOmegaPi::GETTER): { \
+    inputFeatures.emplace_back(candidate.GETTER());                  \
+    break;                                                           \
+  }
+
+// Variation of CHECK_AND_FILL_VEC_OMEGAC0_FULL(OBJECT, FEATURE, GETTER)
+// where GETTER is a method of hfHelper
+#define CHECK_AND_FILL_VEC_OMEGAC0_HFHELPER(OBJECT, FEATURE, GETTER)  \
+  case static_cast<uint8_t>(InputFeaturesOmegacToOmegaPi::FEATURE): { \
+    inputFeatures.emplace_back(hfHelper.GETTER(OBJECT));              \
+    break;                                                            \
+  }
+
+// Variation of CHECK_AND_FILL_VEC_OMEGAC0_HFHELPER(OBJECT, FEATURE, GETTER)
+// where GETTER1 and GETTER2 are methods of hfHelper, and the variable
+// is filled depending on whether it is a OMEGAC0 or a OMEGAC0bar
+#define CHECK_AND_FILL_VEC_OMEGAC0_HFHELPER_SIGNED(OBJECT, FEATURE, GETTER1, GETTER2) \
+  case static_cast<uint8_t>(InputFeaturesOmegacToOmegaPi::FEATURE): {                 \
+    if (pdgCode == o2::constants::physics::kOmegaC0) {                                \
+      inputFeatures.emplace_back(hfHelper.GETTER1(OBJECT));                           \
+    } else {                                                                          \
+      inputFeatures.emplace_back(hfHelper.GETTER2(OBJECT));                           \
+    }                                                                                 \
+    break;                                                                            \
+  }
+
+namespace o2::analysis
+{
+enum class InputFeaturesOmegacToOmegaPi : uint8_t {
+  xPv = 0,
+  yPv,
+  zPv,
+  xDecayVtxCascade,
+  yDecayVtxCascade,
+  zDecayVtxCascade,
+  xDecayVtxV0,
+  yDecayVtxV0,
+  zDecayVtxV0,
+  signDecay,
+  covVtxCharmBaryon0,
+  covVtxCharmBaryon1,
+  covVtxCharmBaryon2,
+  covVtxCharmBaryon3,
+  covVtxCharmBaryon4,
+  covVtxCharmBaryon5,
+  pxCharmBaryon,
+  pyCharmBaryon,
+  pzCharmBaryon,
+  pxCasc,
+  pyCasc,
+  pzCasc,
+  pxPiFromCharmBaryon,
+  pyPiFromCharmBaryon,
+  pzPiFromCharmBaryon,
+  pxLambda,
+  pyLambda,
+  pzLambda,
+  pxKaFromCasc,
+  pyKaFromCasc,
+  pzKaFromCasc,
+  pxPosV0Dau,
+  pyPosV0Dau,
+  pzPosV0Dau,
+  pxNegV0Dau,
+  pyNegV0Dau,
+  pzNegV0Dau,
+  impactParCascXY,
+  impactParPiFromCharmBaryonXY,
+  errImpactParCascXY,
+  errImpactParPiFromCharmBaryonXY,
+  invMassLambda,
+  invMassCascade,
+  invMassCharmBaryon,
+  cosPAV0,
+  cosPACharmBaryon,
+  cosPACasc,
+  cosPAXYV0,
+  cosPAXYCharmBaryon,
+  cosPAXYCasc,
+  ctauOmegac,
+  ctauCascade,
+  ctauV0,
+  etaV0PosDau,
+  etaV0NegDau,
+  etaKaFromCasc,
+  etaPiFromCharmBaryon,
+  etaCharmBaryon,
+  etaCascade,
+  etaV0,
+  dcaXYToPvV0Dau0,
+  dcaXYToPvV0Dau1,
+  dcaXYToPvCascDau,
+  dcaCascDau,
+  dcaV0Dau,
+  dcaCharmBaryonDau,
+  decLenCharmBaryon,
+  decLenCascade,
+  decLenV0,
+  errorDecayLengthCharmBaryon,
+  errorDecayLengthXYCharmBaryon,
+  ptCharmBaryon,
```

Comment:
I deleted HfMlResponseOmegacToOmegaPi.h. I'm now trying to follow the omegac0 implementation of machine learning in Run 2 and make a first attempt in Run 3. I haven't actually identified which variables will be used in machine learning yet, so I've added all the variables that might be used to the header.
I think it would be better to add the machine learning header after analysing it locally and identifying the variables I need, so I'm deleting it now.